matrix-sdk-android/src/main/java/org/matrix/android/sdk/internal/network/Request.kt

```diff
@@ -28,22 +29,21 @@ import java.io.IOException
 
 /**
  * Execute a request from the requestBlock and handle some of the Exception it could generate
+ * Ref: https://github.com/matrix-org/matrix-js-sdk/blob/develop/src/scheduler.js#L138-L175
```

Comment:
I think it should now point to this url @bmarty: https://github.com/matrix-org/matrix-js-sdk/blob/develop/src/scheduler.ts#L54-L92